### PR TITLE
Update progress step

### PR DESCRIPTION
**Pull request by evalouderback**
#### Screens

Screen Shot 2020-08-10 at 3 54 32 PM

#### Breaking Changes

_Please reflect if your changes may break in some way (changes like removing/renaming props are examples of breaking changes)_

#### Checklist:

- [X] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [X] **SCREENSHOT** Please add a screen shot or two
- [X] **SPECS** Please cover your changes with specs
- [X] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

File: app/pb_kits/playbook/pb_progress_step/_progress_step.scss [hunk 1]
```diff
@@ -219,6 +235,167 @@ $pb_progress_step_connector_height_from_left: $pb_progress_step_circle_size/2 -
...
+        [class*=pb_progress_step_item_kit] {
+            &[class*=_complete] {
+                flex-basis: 100%;
+                [class=pb_progress_step_item_wrapper] {
+                    @include pb_progress_step_item_wrapper_child;
+                    [class^=pb_progress_step_item] {
+                        &[class*=_connector_first], &[class*=_connector_second] {
+                            @include pb_progress_step_line($space_sm, $pb_progress_step_connector_width, $primary, $pb_progress_step_connector_height_from_top 0 0 0);
+                        }
+                    }
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_progress_step/_progress_step.scss:265]:
```scss
@mixin progress_step_item_combined($connector1: "_connector_first", $connector2: "_connector_second", $step_circle) {
  [class^=pb_progress_step_item] {
    &[class*=#{$connector1}], &[class*=#{$connector2}] {
      @content
    }
    @if $step_circle {
      &[class*=_step_circle] {
         @include pb_progress_step_circle($primary, $primary, 2px, $primary);
       }
    }
  }
}

@mixin progress_step_item($connector: "_connector_first", $step_circle) {
  [class^=pb_progress_step_item] {
    &[class*=#{$connector1}] {
      @content
    }
    @if $step_circle {
      &[class*=_step_circle] {
         @include pb_progress_step_circle($primary, $primary, 2px, $primary);
       }
    }
  }
}
```

Useage:

```scss
@include progress_step_item_combined {
  @include pb_progress_step_line($space_sm, $pb_progress_step_connector_width, $primary, $pb_progress_step_connector_height_from_top 0 0 0);
}
```

File: app/pb_kits/playbook/pb_progress_step/_progress_step.scss [hunk 2]
```diff
@@ -219,6 +235,167 @@ $pb_progress_step_connector_height_from_left: $pb_progress_step_circle_size/2 -
...
+                }
+            }
+            &[class*=_active] {
+                flex-basis: 100%;
+                [class=pb_progress_step_item_wrapper] {
+                    @include pb_progress_step_item_wrapper_child;
+                    [class^=pb_progress_step_item] {
+                        &[class*=_connector_first] {
+                            @include pb_progress_step_line($space_sm, $pb_progress_step_connector_width, $primary, $pb_progress_step_connector_height_from_top 0 0 0);
+                        }
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_progress_step/_progress_step.scss:289]:
```scss
@mixin progress_step_item($connector: "_connector_first", $step_circle) {
  [class^=pb_progress_step_item] {
    &[class*=#{$connector1}] {
      @content
    }
    @if $step_circle {
      &[class*=_step_circle] {
         @include pb_progress_step_circle($primary, $primary, 2px, $primary);
       }
    }
  }
}


@include progress_step_item {
  @include pb_progress_step_line($space_sm, $pb_progress_step_connector_width, $primary, $pb_progress_step_connector_height_from_top 0 0 0);
}
```

File: Gemfile.lock [hunk 3]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    playbook_ui (6.0.0)
+    playbook_ui (6.0.1)
```
Comment (jasoncypret) [inline Gemfile.lock:4]: You should rebase off the lastest in master

**Comment by jasoncypret** [comment]
@evalouderback  Did you rebase or merge? This says you have 205 changed files. I think your rebase/merge went wrong

**Comment by evalouderback** [comment]
@jasoncypret I think I know what might have gone wrong. Should I manually delete them on github or is there a better way?

**Comment by jasoncypret** [comment]
I would roll back to the last good commit `git reset --hard 03ffb71` 03ffb71 I think? Once you are reset try your `git merge origin/v6.2.0`. Be sure and force with lease instead of pushing

**Comment by jasoncypret** [comment]
@evalouderback I think I just fixed your problem. I changed your base branch to 6.2.0. It was set to 6.1.0

**Pull request merged**